samples/Calling/src/app/views/CallScreen.tsx

```diff
@@ -108,7 +108,19 @@ const TeamsCallScreen = (props: TeamsCallScreenProps): JSX.Element => {
     throw new Error('A MicrosoftTeamsUserIdentifier must be provided for Teams Identity Call.');
   }
 
-  const adapter = useTeamsCallAdapter({ ...adapterArgs, userId, locator }, afterCreate);
+  const adapter = useTeamsCallAdapter(
+    {
+      ...adapterArgs,
+      userId,
+      locator,
+      options: {
```

Comment:
does this need memoized?

Reply:
That's a good catch, and this shouldn't be checked into the PR, this is just for test code

Reply:
Reverted